web/styles/app_variables.css

```diff
@@ -495,7 +498,6 @@
         hsl(0deg 0% 11%)
     );
     --color-background-unread-counter-dot: hsl(105deg 2% 50% / 90%);
-    --color-background-unread-counter-popover-menu: hsl(105deg 2% 50% / 50%);
```

Comment:
These removals should be in the commit that removes that icon use, I think?

Reply:
Fixed. Sorry for the mistake, I think I made the error while squashing commits.